test/unit/beacon_api/beacon_api_v2_test.exs

```diff
@@ -0,0 +1,78 @@
+defmodule Unit.BeaconApiTest.V2 do
+  use ExUnit.Case
+  use Plug.Test
+  use Patch
+
+  alias BeaconApi.Router
+  alias LambdaEthereumConsensus.Store.BlockDb
+  alias LambdaEthereumConsensus.Store.Blocks
+  alias LambdaEthereumConsensus.Store.Db
+
+  @moduletag :beacon_api_case
+
+  @opts Router.init([])
+
+  setup do
+    start_supervised!(Blocks)
+    :ok
+  end
+
+  test "get block by id" do
+    start_link_supervised!({Db, dir: "tmp/beacon_api_v2/1"})
```

Comment:
We could use [the `:tmp_dir` tag functionality from `ExUnit`](https://hexdocs.pm/ex_unit/main/ExUnit.Case.html#module-tmp-dir). Also, we should add this functionality to the original `v1` tests.